js/ad_manager_ooyala.js

```diff
@@ -0,0 +1,207 @@
+/**
+ * Ad Manager For Ooyala video as Ads
+ */
+require("../html5-common/js/utils/InitModules/InitOOHazmat.js");
+require("../html5-common/js/utils/constants.js");
+require("../html5-common/js/utils/environment.js");
+
+OO.Ads.manager(function(_, $) {
+  /**
+   * @class OoyalaAdManager
+   * @classDesc The Ooyala Ads Manager class, registered as an ads manager with the ad manager controller.
+   * Controls how Ooyala videos loaded and played as ads while communicating with the ad manager framework.
+   * @public
+   * @property {string} name Name of the OoyalaAdManager, must match what is sent from Backlot and used at the page level
```

Comment:
We only need these tags for the public variables (this.xx is public but var xx is private)